### ISSUE TITLE
Messages not marked as read are also not marked as delivered

### ISSUE THREAD

**Issue by iomintz** [lwesterhof/semaphore#5]
```py
def handler(ctx):
    pass

bot.register_handler('', handler)
```
this code doesn't even mark the message as *delivered*, which is bad UX because the user can't tell if the bot is down or it just decided not to respond. IMO, the bot should mark as read on response, and mark as delivered always.

**Comment by lwesterhof** [comment]
Will fix as soon as this is implemented in signald: https://gitlab.com/thefinn93/signald/-/issues/64

**Comment by lwesterhof** [comment]
Fixed in 437bd74.
